### ISSUE TITLE
Didn't work any more !

### ISSUE THREAD

**Issue by imeDevelopers** [rmtmckenzie/flutter_qr_mobile_vision#163]
the plugin didn't work any more ..
the callback function not triggered ...

tried many times, different qr codes , but same results ..
plugin versoin: 2.0.0
test device: Realme 5
android: 10
flutter Version: 1.22.5 stable
dev system: win 10
android studio: 4.1.1
gradle: 6.1.1
compilesdk: 30
targetsdk: 30

**Comment by crcaicedo** [comment]
Same problem here, not working at all

plugin versoin: 2.0.0
test device: Moto G6+
android: 9
flutter Version: 1.22.5 stable
dev system: MacOs BigSur
android studio: 4.1.1
gradle: 5.4.1
compilesdk: 29
targetsdk: 29

**Comment by GabrieleVolpato** [comment]
I'm facing the same problem.

plugin versoin: 2.0.0
test device: Mi A3
android: 10
flutter Version: 1.22.5 stable
dev system: MacOs BigSur
android studio: 4.1.1
gradle: 6.7
compilesdk: 30
targetsdk: 30

**Comment by jasondelport** [comment]
Hi,

I was able to fix this issue by bumping the gradle dependency version of MLKit.

Go to:

/Users/{username}/.pub-cache/hosted/pub.dartlang.org/qr_mobile_vision-2.0.0/android (on mac)

open build.gradle and change mlkit dependency to latest version.

dependencies {
    implementation 'com.google.mlkit:barcode-scanning:16.1.0'
}

**Comment by itsJoKr** [comment]
Having similar issue, might be connected: 

- Samsung S7 - everything works as expected
- OnePlus 6 - doesn't work. Was able to scan one barcode and 0 QR, data matrix codes.

Bumping MLKit version did not fix it.

**Comment by TercyoStorck** [comment]
Found a workaround for android. Go to /android/build.gradle and add this to your dependencies:

`implementation 'com.google.mlkit:barcode-scanning:16.1.0'`

**Comment by Manuel-Indlekofer** [comment]
This seems to be device/screen size related. While working fine on a large variety of phones, it was not working on a pixel 5. I think it is related to the size of the preview, beacause it did work on the pixel 5, after changing the dimensions of the camera preview

**Comment by gyenesandras** [comment]
I'm facing the same issue. Barcode scanning works fine on many Android phones but it does not work on a Pixel 4.

@Manuel-Indlekofer, what did you mean by changing the size of the preview? Did you change the size of the QrCamera widget to a static value?

**Comment by Manuel-Indlekofer** [comment]
@gyenesandras yes. For testing purposes i set the preview to a fixed size of 300x600. This worked fine on the pixel5. I also had success using the suggested dependency

> `implementation 'com.google.mlkit:barcode-scanning:16.1.0'`

**Comment by gyenesandras** [comment]
> @gyenesandras yes. For testing purposes i set the preview to a fixed size of 300x600. This worked fine on the pixel5. I also had success using the suggested dependency
> 
> > `implementation 'com.google.mlkit:barcode-scanning:16.1.0'`

@Manuel-Indlekofer, thank you!

**Comment by andriesatya** [comment]
Hi, any idea how to reproduce this in the emulator? I've tried to make an emulator with screen size 480 x 800, but still can't reproduce the issue.

**Comment by imeDevelopers** [comment]
Hi all, i found a solution long time ago and the plugin works, but i forgot this open issue. so i currently writing this comment when i saw this thread.

Just change the size of camera widget to 600X600 and it will work ...

But the QrCamera() widget inside Container() widget with height 600

and it will work, thank you.
